app/merginapi.cpp

```diff
@@ -53,6 +53,8 @@ MerginApi::MerginApi( LocalProjectsManager &localProjects, QObject *parent )
 
   loadAuthData();
   GEODIFF_init();
+  GEODIFF_setLoggerCallback( &GeodiffUtils::log );
+  GEODIFF_setMaximumLoggerLevel( GEODIFF_LoggerLevel::LevelInfo );
```

Comment:
Just checked, python client uses LevelDebug all the time, let's use it here as well - better more logging than not enough :-)